.ci/commitlint.config.ts

```diff
@@ -0,0 +1,70 @@
+import type {UserConfig} from '@commitlint/types';
+
+const Configuration: UserConfig = {
+  /*
+   * Resolve and load @commitlint/config-conventional from node_modules.
+   * Referenced packages must be installed
+   */
+  extends: ['@commitlint/config-conventional'],
+  /*
+   * Resolve and load @commitlint/format from node_modules.
+   * Referenced package must be installed
+   */
+  formatter: '@commitlint/format',
+  /*
+   * Any rules defined here will override rules from @commitlint/config-conventional
+   */
+  rules: {
+    "body-full-stop": [0, "always", '.'],
+    "body-leading-blank": [2, "always"],
+    "body-empty": [0, "never"],
+    "body-min-length": [2, "always", 1],
+    "body-case": [0, "never", "lower-case"],
+    "footer-leading-blank": [2, "always"],
+    "footer-empty": [0, "never"],
+    "footer-max-length": [0, "always", 72],
+    "header-case": [0, "never", "lower-case"],
+    "header-full-stop": [2, "never", "."],
+    "header-max-length": [2, "always", 72],
+    "header-min-length": [2, "always", 1],
+    "references-empty": [0, "never"],
+    "scope-case": [2, "always", "lower-case"],
+    "subject-case": [0, "never", "lower-case"],
+    "subject-empty": [0, "never"],
+    "subject-full-stop": [2, "never", "."],
+    "signed-off-by": [2, "always", "Signed-off-by:"],
+    /*
+    * Enable type if we need, warn it currently.
+    */
+    "type-enum": [0, "always", ['ci', 'docs', 'feat', 'fix', 'perf', 'refactor', 'revert', 'rfe', 'style']],
+    "type-case": [2, "always", "lower-case"],
```

Comment:
The `type-enum` is disabled.  Do we need enable the rule for `type-case`  or user can use any type they want  ?

Reply:
Currently it's enabled, see commit 2, and I think using lower-case type should be better, but we can disable as well if other peoples agree.

Reply:
Yes~   I see it is enabled now.  Let me make myself more clearly ~
The `type-enum` be disabled then use can use any `type`  (and same with `scope`).  However the `type-case` and `scope-case` check be enabled. And i notice there are some usage about the upper case before - `IPU:`  `PPC:`, `HPT:`.   IMO, it is acceptable, low-case is not must ^^

I prefer to remove this rule.

Welcome for other opinions ~  Thanks!